public/tailwind.css

```diff
@@ -2111,4 +2111,116 @@ video {
     padding-left: 0px;
     padding-right: 0px;
   }
+}
+
+.mission-background,
+.sanctify-screen-time-background,
+.technology-art-collide-background,
+.what-people-saying-background,
+.support-mission-background {
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left center;
+  height: 700px;
```

Comment:
Can you add these using the tailwind classes? This file gets overwritten, it is generated by tailwind.